runtime/ops/bootstrap.rs

```diff
@@ -42,11 +42,20 @@ pub struct SnapshotOptions {
 
 impl Default for SnapshotOptions {
   fn default() -> Self {
+    let arch = std::env::consts::ARCH.to_owned();
+    let platform = std::env::consts::OS.to_owned();
```

Comment:
Is it absolutely necessary to call `.to_owned()` here?

Reply:
Good point, both can be removed here.